### ISSUE TITLE
Messages component - a11y alert

### ISSUE THREAD

**Issue by anqaka** [SnowdogApps/magento2-alpaca-components#495]
Notification message is not read by screen reader when it dynamically appears on the page.
I suppose that the content which is dynamically loaded should be inside container with `aria-live="polite"` - to investigate and improve
